win32/src/winapi/types.rs

```diff
@@ -40,3 +44,40 @@ impl<'a> super::stack_args::FromStack<'a> for POINT {
         POINT { x, y }
     }
 }
+
+// https://learn.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-
+pub const ERROR_FILE_NOT_FOUND: u32 = 2;
```

Comment:
Possibly some `enum ERROR { FILE_NOT_FOUND = 2 }` maybe?
I guess we need to be able to handle arbitrary u32 errors though, hrm.

Reply:
Extra conversions to/from an ERROR enum just feel unnecessary, personally. Especially given that we’ll be dealing with arbitrary u32s, like you mentioned.